doc/benchmark/modules/ROOT/pages/scenario/authorization-code.adoc

```diff
@@ -47,7 +47,7 @@ bin/kcb.sh \
 
 == Variants
 
-To create offline sessions, set the parameter xref:configuration.adoc#scope[`--scope`] to a value including `offline_token`, for example, `openid profile offline_token`.
+To create offline sessions, set the parameter xref:configuration.adoc#scope[`--scope`] to a value including `offline_token`, for example, `openid profile, offline_token`.
```

Comment:
Are you sure, that the comma at the end is correctly? To me it seems like a multi word subject then a list.

Reply:
@lhanusov - the `openid profile offline_token` is the scope, and multiple value (three in this case) need to passed in separated as blanks. It is part of the OIDC API, see https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest. 

Therefore, it needs to be as is.

Reply:
you both are right about it, it was a change I had a question around as well. But its good to know. I will revert my change.

Reply:
@ahus1 so in other words is this something a user should expect to run for the offline_token session run ? Would it be better to update the help doc to reflect this, as I didnt knew on the first attempt that it should be this way.

```
./kcb.sh --scenario=keycloak.scenario.authentication.LoginUserPassword \
--server-url=http://keycloak.$(minikube ip).nip.io --realm-name=test-realm \
--client-redirect-uri=* --scope=openid profile offline_token
```

Reply:
We could extend the section to have more details on that, +1 on this. 

At the same time offline tokens is not the standard use case, therefore the first example should stay the way it is. 

In order to make this work, one would need to add some quotes around the scope, so that the CLI knows they are one value for the scope. I have to admit I called this via the IDE and a system property, not via the CLI. It sould be something like the following, and it needs to be tested before adding it to the docs:

```
--scope="openid profile offline_token"
```

I see you put "*" as a client redirect URI there. While `*` is a valid configuration when configuring the client (it is a wildcard), a `*` is not a valid redirect URI as it doesn't start with `http://...` or `https://...` ... not sure what kind of validations are there.

Reply:
This is what I am getting if I am using with and with out quotes

```
$ ./kcb.sh   --scenario=keycloak.scenario.authentication.AuthorizationCode --server-url=http://keycloak.$(minikube ip).nip.io   --realm-name=test-realm --scope="openid profile offline_token"
Error: Could not find or load main class profile
Caused by: java.lang.ClassNotFoundException: profile

$ ./kcb.sh   --scenario=keycloak.scenario.authentication.AuthorizationCode --server-url=http://keycloak.$(minikube ip).nip.io   --realm-name=test-realm --scope=openid profile offline_token
07:47:06.014 [INFO ] a.e.s.Slf4jLogger - Slf4jLogger started
==========================================================
Target servers: [http://keycloak.192.168.39.218.nip.io]
Scenario:
 Name: keycloak.scenario.authentication.AuthorizationCode
```

also is this the right query on a PostgreSQL DB to verify the presence of offline session tokens ? `select * from kc_user_session where offline=true;` If yes, I dont see it getting populated by the run which didnt use the double quotes.

Reply:
I had a look and the `kcb.sh` doesn't pass on the quotes to the Java CLI. Even with the second variant the two second parameters are lost, therefore there is no offline token created. You can see what is happening when adding `set -x` to the script temporarily. 

I suggest to create a new issue to enhance the `kcb.sh` script to support this, and to keep this out of this PR.

Reply:
Makes sense, I will create a separate issue for that and revert my original change.

Reply:
https://github.com/keycloak/keycloak-benchmark/issues/204 created this for the parsing issue by the `kcb.sh`